src/card/card.styles.js

```diff
@@ -0,0 +1,61 @@
+import { css } from "lit-element";
+
+export default css`
+  .body-search { 
+      display: flex;
+      justify-content: center;
+    }
+    
+  .list-wrapper {
+    align-items: center;
+    display: flex;
+    flex-grow: 1;
+    justify-content: center;
+    width: 100%;
+  }
+
+  .list-cards {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    max-width: 936px;
+    width: 100%;
+  }
+
+  .list-card {
+    background: white;
+    border: solid;
+    box-shadow: 4px 4px 10px 0 rgba($black, 0.5);
+    display: flex;
+    flex-flow: column;
+    margin: 16px;
+    max-height: 300px;
+    max-width: 198px;
+    padding: 28px 28px 24px;
+    transition: transform 0.2s;
+
+    &:hover {
+      transform: scale(1.1);
+    }
```

Comment:
You aren't using sass, I think this doesn't work 🤔

Reply:
Done!